src/Bitcoin/FeeCalculator.cpp

```diff
@@ -21,6 +21,15 @@ int64_t DefaultFeeCalculator::calculateSingleInput(int64_t byteFee) const {
     return int64_t(148) * byteFee;
 }
 
+int64_t SegwitFeeCalculator::calculate(int64_t inputs, int64_t outputs, int64_t byteFee) const {
+    const auto txsize = int64_t(std::ceil(101.25 * inputs + 31.0 * outputs + 10.0));
+    return txsize * byteFee;
+}
+
+int64_t SegwitFeeCalculator::calculateSingleInput(int64_t byteFee) const {
+    return int64_t(std::ceil(101.25)) * byteFee;
```

Comment:
just use a const `102` to remove all `std::ceil` calls

Reply:
Modern compilers are smart enough to replace this to 102 at compile time, but I changed as you suggest to be sure :)